src/esm_runscripts/filelists.py

```diff
@@ -901,11 +901,43 @@ def resolve_symlinks(config, file_source):
 
 
 def copy_files(config, filetypes, source, target):
+    """
+    This function has a misleading name. It is not only used for copying, but also
+    for moving or linking, depending on what was specified for the particular file
+    or file type vie the ``file_movements``.
+
+    Note: when the ``target`` is ``thisrun`` (intermediate folders) check whether the
+    type of file is included in ``intermediate_movements``. If it's not, instead of
+    moving the file to the intermediate folder it moves it to ``work``. This is an
+    ugly fix to provide a fast solution to the problem that files are
+    copied/moved/linked twice unnecessarily, and this affects inmensely the performance
+    of high resolution simulations. A better fix is not made because ``filelists`` are
+    being entirely reworked, but the fix cannot wait.
+
+    Parameters
+    ----------
+    config : dict
+        The general configuration
+    filetypes : list
+        List of file types to be copied/linked/moved
+    source : str
+        Specifies the source type, to be chosen between ``init``, ``thisrun``,
+        ``work``.
+    target : str
+        Specifies the target type, to be chosen between ``init``, ``thisrun``,
+        ``work``.
+    """
     logger.debug("\n::: Copying files")
     helpers.print_datetime(config)
+
     successful_files = []
     missing_files = {}
 
+    intermediate_movements = config["general"].get(
+        "intermediate_movements",
+        [],
+    )
+
```

Comment:
This is probably trivial, but is a bit confusing. If there is no defined intermediate movements by default, you get an empty list. Above, in `configs/defaults/general.yaml` is where the actual default list is defined. Maybe put it here instead? Or just add a comment so it is clearer:

```suggestion
    # See the default intermediate movements list in `configs/defaults/general.yaml`
    intermediate_movements = config["general"].get(
        "intermediate_movements",
        [],
    )

```